packages/grafana-ui/src/components/Card/CardContainer.tsx

```diff
@@ -123,9 +133,18 @@ export const getCardContainerStyles = (
       position: 'relative',
       pointerEvents: disabled ? 'none' : 'auto',
       marginBottom: theme.spacing(1),
-      transition: theme.transitions.create(['background-color', 'box-shadow', 'border-color', 'color'], {
-        duration: theme.transitions.duration.short,
-      }),
+      ...handleReducedMotion(
+        {
+          transition: theme.transitions.create(['background-color', 'box-shadow', 'border-color', 'color'], {
+            duration: theme.transitions.duration.short,
+          }),
```

Comment:
Hmmm. 

I wonder if instead of `handleReducedMotion`  (or as well as) there's something we could add/change to this `theme.transitions.create` thing to handle this. Unsure.